tests/test_idutils.py

```diff
@@ -866,3 +866,10 @@ def test_ascl():
     assert idutils.is_ascl("ascl:1908.011")
     assert idutils.is_ascl("ascl:1908.0113")
     assert not idutils.is_ascl("1990.0803")
+
+
+def test_url():
+    """Test URL validation."""
+    assert idutils.is_url(
+        "https://archive.softwareheritage.org/swh:1:dir:44ac666e75004dd2a27ca0e09e73aecc0e8b426f;origin=https://github.com/amykwebster/MIPseq_2021;visit=swh:1:snp:46bcbe75339295c3d68fcf1257022aae8e6bee40;anchor=swh:1:rev:27839dcc9ef1587086be195349310fb70fbfcaf1"
```

Comment:
Can you also add an example above in the big list of sample `identifiers` variable? I have a feeling that this should eventually also be [identified as a `swh` identifier](https://github.com/inveniosoftware/idutils/blob/08bd7b6921febd90dfa64e8eb8d4eecab5fd9c73/idutils/__init__.py#L984).

We can do this later of course by including the `http(s)://archive.softwareheritage.org` part in the regex.

Reply:
swh was actually already in the identifiers list, but with a url that would have failed the validation check. I expanded the test so all the urls in the identifiers variable are now checked.